crates/burn-import/src/burn/node/prelu.rs

```diff
@@ -12,17 +12,16 @@ impl NodeCodegen for onnx_ir::prelu::PReluNode {
 
     fn field(&self) -> Option<Field> {
         let name = Ident::new(&self.name, Span::call_site());
+        let num_parameters = self.config.num_parameters;
 
-        Some(Field::new(
-            self.name.clone(),
-            quote! {
-                PRelu<B>
-            },
-            quote! {
-                let #name = PReluConfig::new()
-                    .init(device);
-            },
-        ))
+        let init = if num_parameters > 1 {
+            let n = num_parameters.to_tokens();
+            quote! { let #name = PReluConfig::new().with_num_parameters(#n).init(device); }
+        } else {
+            quote! { let #name = PReluConfig::new().init(device); }
+        };
```

Comment:
Always init with `with_num_parameters(#n)` and don't rely on `PReluConfig`'s defaults so we have a single source.